pkg/cli/options_test.go

```diff
@@ -34,6 +35,65 @@ import (
 )
 
 var _ = Describe("Discover external plugins", func() {
+	Context("validate plugins root path", func() {
+		var (
+			homePath   string = os.Getenv("HOME")
+			customPath string = "/usr/local/myplugins"
+			xdghome    string = os.Getenv("XDG_CONFIG_HOME")
+		)
+
+		It("based on the host os", func() {
```

Comment:
Something to consider here is how maintainable and readable these tests are. Often times reading through test cases can help someone get an understanding of the general flow of what is being tested. 

In this particular case, when I look at this from the perspective of someone who has no idea what the logic the test is for this takes me a bit longer to read through the code and understand what is happening. 

My recommendation is to look into using the various Gingko functions to break these tests into much smaller tests. Something like:
```go
...
When("using the $HOME environment variable", func{
...
  It("should return the correct path for the darwin OS", func{...})
  It("should return the correct path for the linux OS", func{...})
...
})
...
```
Following this approach we can make the individual tests significantly smaller and easier to follow, thus making them much easier to maintain.